test/python/test_set_backend.py

```diff
@@ -19,54 +19,68 @@
 import tensorflow as tf
 import ngraph_bridge
 
+from common import NgraphTest
+
 
 # Test ngraph_bridge config options
-def test_set_backend():
-    ngraph_bridge.enable()
-    backend_cpu = 'CPU'
-    backend_interpreter = 'INTERPRETER'
+class TestSetBackend(NgraphTest):
+
+    def test_set_backend(self):
+        # store env variables
+        # when testing on backends like GPU the tests are run with NGRPAH_TF_BACKEND
+        # by storing and restoring the env_variables we run the tests independent of the backend set
+        # currently we only and restore only NGRPAH_TF_BACKEND
```

Comment:
typo: ` # currently we only and restore only NGRPAH_TF_BACKEND` ---> ` # currently we only restore NGRPAH_TF_BACKEND`